src/load-secrets/types.ts

```diff
@@ -11,13 +11,16 @@ export interface JsonSecretDescribedField {
   example?: string
 }
 
+export interface StringSecret extends BaseSecret {
```

Comment:
nit: Could we have some documentation on this interface to describe the use case?